docs/api/utils/index.rst

```diff
@@ -0,0 +1,36 @@
+.. _utils_api :
+
+qcodes.utils
+============
+
+
+.. autosummary::
+
+    qcodes.utils
+    qcodes.utils.command
+    qcodes.utils.deprecate
+    qcodes.utils.helpers
+    qcodes.utils.magic
+    qcodes.utils.metadata
+    qcodes.utils.plotting
+    qcodes.utils.threading
+    qcodes.utils.validators
+    qcodes.utils.zmq_helpers
+
+.. automodule:: qcodes.utils
+
+
+.. toctree::
+   :maxdepth: 4
+   :hidden:
+
+   command
+   deprecate
+   helpers
+   magic
+   metadata
+   plotting
+   slack
+   threading
```

Comment:
so what are the steps for the developer once he adds a new python module that deserves being included in the docs? perhaps, those can be added to "contributing" or wherever?

Reply:
Yes that would be great once that is rewritten.

Reply:
can a rudimentary description be a part of this PR?

Reply:
Yes, but the problem is not the description. The problem is that it would land in the middle of another document where the rest no longer applies

Reply:
i'd be ok with that because without the description i'd have to bother you when i make "my next PR" :) Full-blown proper fixing of that document can be done later.

Reply:
Ok I guess it should go here https://qcodes.github.io/Qcodes/community/contributing.html#development ?

Reply:
good! thanks!